CommonTools/RecoAlgos/interface/RecoTrackSelectorBase.h

```diff
@@ -19,6 +19,8 @@ class RecoTrackSelectorBase {
     ptMin_(cfg.getParameter<double>("ptMin")),
     minRapidity_(cfg.getParameter<double>("minRapidity")),
     maxRapidity_(cfg.getParameter<double>("maxRapidity")),
+    meanPhi_((cfg.getParameter<double>("minPhi")+cfg.getParameter<double>("maxPhi"))/2.),
+    rangePhi_((cfg.getParameter<double>("maxPhi")-cfg.getParameter<double>("minPhi"))/2.),
```

Comment:
@fwyzard I agree. The idea is that `minPhi` should be smaller than `maxPhi` (I can certainly add a check for that), and in case one needs to specify a range containing π (which I understood was your point), the `maxPhi` is set to a value larger than π (I could also add checks that `minPhi` must be smaller than π, and `maxPhi` must be larger than -π).

The range from 3π/2 to π/2 (going through π) can be specified by reversing your `minPhi` and `maxPhi`
```
minPhi = 1.571    # 1/2 π
maxPhi = 4.712    #  3/2 π
```
and then it should work (selecting tracks whose phi is in range `3.1415 +- 1.5705`).

I can also add some documentation to the classes to clarify the intended usage (as it was a bit tricky and took me some time end up with this implementation).